css/compositing/parsing/background-blend-mode-computed.html

```diff
@@ -29,6 +29,15 @@
 test_computed_value("background-blend-mode", "color");
 test_computed_value("background-blend-mode", "luminosity");
 
+// per spec, excess values should be dropped.
```

Comment:
Maybe "Per spec, excess values should not be used, but that doesn't affect the computed style" or something.

Reply:
fixed.